experiments/functional/backup_and_restore/induce_failure.yml

```diff
@@ -0,0 +1,26 @@
+---
+- block:
+   
+   - name: Getting the volume name
+     shell: kubectl get pvc {{ app_pvc }} -n {{ app_ns }} -o jsonpath='{.spec.volumeName}'
+     register: vol_name
+     
+   - name: Getting the CVR name from the corresponding PV
+     shell:  kubectl get cvr -n openebs -l openebs.io/persistent-volume={{ vol_name.stdout }} -o jsonpath='{range.items[*]}{.metadata.labels.cstorpool\.openebs\.io/name}{"\n"}{end}' | head -2
+     register: cvr
+
+   - name: Getting the pool pod name
+     shell: kubectl get pod -n openebs | grep {{ item }} | awk '{print $1}'
```

Comment:
CSP name is used in pool pod labels. can we use it to derive respective pool pods